ext/openAPI/30125-data-combining.yaml

```diff
@@ -0,0 +1,271 @@
+      operationId: getCombinerMappings
+      tags:
+        - Combiners
+      parameters:
+        - $ref: '#/components/parameters/CombinerId'
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DataCombiningList'
+          description: Success
+        "404":
+          content:
+            application/json:
+              schema:
+                $ref: '../hivemq-edge-openapi-2025.3-SNAPSHOT.yaml#/components/schemas/ProblemDetails'
+          description: Combiner not found
+  /api/v1/management/combiners/{combinerId}/mappings/{mappingId}/instructions:
+    get:
+      description: Get all the instructions for a designated mapping
+      summary: Get all instructions
+      operationId: getMappingInstructions
+      tags:
+        - Combiners
+      parameters:
+        - $ref: '#/components/parameters/CombinerId'
+        - $ref: '#/components/parameters/MappingId'
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                description: List of instructions to be applied to incoming data
+                items:
+                  $ref: '../hivemq-edge-openapi-2025.3-SNAPSHOT.yaml#/components/schemas/Instruction'
+          description: Success
+        "404":
+          content:
+            application/json:
+              schema:
+                $ref: '../hivemq-edge-openapi-2025.3-SNAPSHOT.yaml#/components/schemas/ProblemDetails'
+          description: Combiner not found
+components:
+  parameters:
+    CombinerId:
+      name: combinerId
+      description: The unique id of the combiner to retrieve.
+      in: path
+      required: true
+      schema:
+        type: string
+        format: uuid
+    MappingId:
+      name: mappingId
+      description: The unique id of the mapping to retrieve.
+      in: path
+      required: true
+      schema:
+        type: string
+        format: uuid
+  schemas:
+    CombinerList:
+      type: object
+      description: The list of Combiner defined in this Edge instance
+      properties:
+        items:
+          type: array
+          items:
+            $ref: '#/components/schemas/Combiner'
+    Combiner:
+      type: object
+      description: A data combiner, bringing tags (adapters) and topic filters (bridges) together for further northbound data mapping
+      properties:
+        id:
+          type: string
+          format: uuid
+          description: The unique id of the data combiner
+        name:
+          type: string
+          description: The user-facing name of the combiner
+        description:
+          type: string
+          description: The user-facing description of the combiner
+        sources:
+          $ref: '#/components/schemas/EntityReferenceList'
+        mappings:
+          $ref: '#/components/schemas/DataCombiningList'
+      required:
```

Comment:
i think at least sources should be required as a combiner without them makes no sense, right?